lib/bootloader.js

```diff
@@ -45,7 +45,12 @@
   // version without applicationCache intercepting us.
   function getShadowIndex(success, failure) {
     var req = new XMLHttpRequest();
-    req.open('GET', 'APPSHELL_PATHappshell.html', true);
+    var path = 'APPSHELL_PATHappshell';
+    var query = window.location.search.match(/revision=([^&#]+)/);
```

Comment:
I understand it to be:

1. Load app with current revision, appcache caches using https://subdomain.ticketsolve.com/checkin/ key
2. Load app with revision param
  * appcache miss due to keys being different one with revision param, one without.
  * no manifest, no DOM swap.

Can confirm behavior on latest chrome and FF desktop. I'll check on iOS9 safari and chrome on android.